### ISSUE TITLE
Support TSV (Tab Separated Value) files

### ISSUE THREAD

**Issue by alexbfree** [hestiaAI/hestialabs-experiences#372]
This was seen with a participant today, the File Explorer could not read the file.

**Comment by alexbfree** [comment]
@streitlua said in standup this may already work. @alexbfree to test on Washington Post data.

**Comment by alexbfree** [comment]
(further testing means this supercedes what i said at the end of standup.

I did some testing and this reveals a few things:

- If the tab separated data ends in .TXT it is not detected as tabular data at all.
- If the tab separated data ends in .TSV it is detected, but not correctly parsed into columns
- Also something I noticed (and I think this is true for CSV too) is that if there are headers but no rows, we don't get to see the headers. This would be useful.

On further investigation this *does* remain high priority - I now see that the participant has two tab separated value files - one nearly empty .TSV but another that is .TXT that does have rich tab-separated data we need to view.

Easy way to generate a test file - enter some data in Excel and export to Tab-separated. Try it as .TXT and .TSV
